docs/auth/README.adoc

```diff
@@ -13,3 +15,136 @@ dependencies {
 
 == Usage
 
+To use the Auth SDK you'll first need to:
+
+* Import the Core module. See link:../core/README.adoc[the Core documentation].
+* Specify the package name of your Android project in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file.
+This is so the browser needs to know where to redirect to after authentication. See link:https://github.com/aerogear/aerogear-android-sdk/blob/master/example/build.gradle[example app build.gradle].
+* Specify the same URI in the "valid redirect URIs" configuration of the Keycloak Client the app is authenticating with.
+
+=== Retrieving the current user
+
+To retrieve the current authenticated user the `AuthService#currentUser` method can be invoked.
+
+[source,java]
+----
+AuthService authService = mobileCore.getInstance(AuthService.class);
+
+AuthServiceConfiguration authServiceConfig = new AuthServiceConfiguration.AuthConfigurationBuilder()
+    .withRedirectUri("org.aerogear.mobile.example:/callback")
+    .allowSelfSignedCertificate(true)
```

Comment:
let's remove it this config for now as it promotes bad practice. We may not keep this API in the future when we have certificate pinning.